corehq/apps/users/models.py

```diff
@@ -1247,7 +1247,8 @@ def create(cls, domain, username, password, email=None, uuid='', date='',
         else:
             couch_user.created_on = datetime.utcnow()
 
-        user_data = kwargs.get('user_data', {})
+        user_data = {'commcare_project': domain}
```

Comment:
is there a reason this isn't called `domain`? we already have confusion around project vs domain vs domain_obj, etc.